Movecraft/build.gradle.kts

```diff
@@ -25,7 +26,11 @@ tasks.shadowJar {
         include(project(":movecraft-v1_18"))
         include(project(":movecraft-v1_20"))
         include(project(":movecraft-v1_21"))
+        include(dependency("co.aikar:acf-paper:0.5.1-SNAPSHOT"))
     }
+
+    relocate("co.aikar.commands", "Movecraft.acf")
+    relocate("co.aikar.locales", "Movecraft.locales")
```

Comment:
We should open up an issue to look into an improved locale method.  Both ACF and Kyori look like they have their own, so we'll have to weigh pros and cons.